src/main/java/seedu/address/model/event/StartTime.java

```diff
@@ -28,28 +30,28 @@ public class StartTime implements Comparable<StartTime> {
     public StartTime(String startTime) {
         requireNonNull(startTime);
         assert !startTime.isBlank();
-        checkArgument(isValidStartTime(startTime), MESSAGE_FORMAT_CONSTRAINTS);
+        checkArgument(isValidStartTimeFormat(startTime), MESSAGE_FORMAT_CONSTRAINTS);
         checkArgument(isValidStartTimeValue(startTime), String.format(MESSAGE_VALUE_CONSTRAINTS, startTime));
-        String [] parsedTime = startTime.split(":", 2);
+        String [] parsedTime = startTime.split(TIME_DELIMITER, 2);
```

Comment:
Extra space after `String`

```suggestion
        String[] parsedTime = startTime.split(TIME_DELIMITER, 2);
```